scripts/dandi_ephys_uploads.py

```diff
@@ -22,50 +40,105 @@ def get_creds():
+    nwb_folder_path = args.nwb_folder_path
+    dandiset_id = args.dandiset_id
     dandi_set = dandi()
     dandi_set.dandi_authenticate()
-    dandi_dataset = dandi_set.get_dandiset(args.dandi_val)
-
-    status_probes = []
-    path = 'sub_' + args.subject_id + '/' 'sub_' + args.subject_id + 'sess_' + args.sess_id + '/' +  'sub_' + args.subject_id + '+sess_' + args.sess_id + '_ecephys.nwb'
-    status = dandi_dataset.iter_upload_raw_asset(
-        args.dandi_file,
-        asset_metadata={
-            'path': path,
-            "dandiset": str(dandi_dataset)})
-    dir = os.path.dirname(args.dandi_file)
-    probes = ["probeA", 'probeB', 'probeC', 'probeD', 'probeE', 'probeF']
-    for i in glob(join(dir, "*" + 'probe' + "*.nwb")):
-        for probe in probes:
-            if probe in i:
-                print(i, probe)
-                status_probes.append(
-                    dandi_dataset.iter_upload_raw_asset(
-                        i,
-                        asset_metadata={
-                            'path': 'sub_' + args.subject_id + '/' 'sub_' + args.subject_id + 'sess_' + args.sess_id + '/' +  'sub_' + args.subject_id + '+sess_' + args.sess_id + '+' + probe + '_ecephys.nwb',
-                            "dandiset": str(dandi_dataset)}))
-    print(list(status))
-    for i in status_probes:
-        print(list(i))
-
-    fb.start(fb.get_creds())
+    dandi_dataset = dandi_set.get_dandiset(dandiset_id)
+    session_id = args.sess_id
+    project_id = args.project_id
+    subject_id = args.subject_id
+  
+
+    set_env()
+    dandiset_path = nwb_folder_path
+    assert os.getenv("DANDI_API_KEY"), (
+        "Unable to find environment variable 'DANDI_API_KEY'. "
+        "Please retrieve your token from DANDI and set this environment variable."
+    )
+
+    # Create the session directory
+    directory_path = os.path.join(nwb_folder_path)
+    dandi_path = os.path.abspath(directory_path)
+    dandi_path_set = directory_path  + "/" + dandiset_id
+    print("PATHS: ", directory_path, dandi_path, dandi_path_set)
+    if not os.path.exists(directory_path):
+        os.makedirs(directory_path)
+        os.makedirs(dandi_path_set)
+
+
+    # Rename the nwb file to include the session id
+    src = os.path.join(nwb_folder_path,  "spike_times.nwb")
+    dst = os.path.join(directory_path, session_id + ".nwb")
+
+    # Move the nwb file to the session directory
+    shutil.move(src, dst)
+    
+
+    dandi_download(urls=r'https://dandiarchive.org/dandiset/' + dandiset_id + '/draft/', output_dir=str(dandi_path), get_metadata=True, get_assets=False)
+    dandi_organize(paths=str(directory_path), dandiset_path=str(dandi_path_set))
+
+    # We need to add ses to file names manually when using dandi_organize
+    organized_nwbfiles = glob(join(directory_path, dandiset_id,  'sub-*' ,  "*.nwb" ))
+    for organized_nwbfile in organized_nwbfiles:
+        file_path = Path(organized_nwbfile)
+        if "ses" not in file_path.stem:
```

Comment:
Wouldn't dandi organize use the session id directly if the nwb file had this string be the core object_id of the dataset?